Formula/neovim.rb

```diff
@@ -78,8 +78,9 @@ def install
   def caveats; <<-EOS.undent
       The Neovim executable is called 'nvim'. To use your existing Vim
       configuration:
-          ln -s ~/.vimrc ~/.nvimrc
-          ln -s ~/.vim ~/.nvim
+          ln -s ~/.vimrc ~/.config/nvim/init.vim
+          ln -s ~/.vim ~/.config/nvim
+          ln -s ~/.vim/colors ~/.config/nvim/colors
```

Comment:
Ack sorry. I just deleted all 3 of the symlinks and ran `ln -s ~/.vimrc ~/.config/nvim/init.vim` and `ln -s ~/.vim ~/.config/nvim` and it seems to be working as we'd like-- with my color scheme. Not sure why it didn't work the first time.